### ISSUE TITLE
We cannot choose standard plan in Pivotal Tile

### ISSUE THREAD

**Issue by yukikashiwabara** [Azure/meta-azure-service-broker#202]
We cannot select Standard plans for Postgres in 1.10.0 version.
Could you check it?

```
azure-postgresqldb           basic1*, basic2*                                                                                                                                                                                                                                                                                                                                                                           Azure Database for PostgreSQL Service
```

**Comment by zhongyi-zhang** [comment]
Hi @yukikashiwabara, there was a pricing model breaking change from Gen4 hardware to Gen5 in Azure PostgreSQL Service. The old pricing model has Basic, Standard, and Premium tiers. While the new model has Basic, General Purpose, and Memory Optimized. FYI: https://azure.microsoft.com/en-us/pricing/details/postgresql/. Now both Gen4 and the old pricing model are deprecated and not supported in most of the Azure regions. And Gen5 doesn't support the old pricing model. (You can verify these on Azure Portal.)
As MASB is in maintenance mode now, we just refactored to support Gen5 and retained only the Basic tier. So what you see is expected. Sorry for the inconvenience...

**Comment by yukikashiwabara** [comment]
Thank you very much for the response.
At least, I understand it is not bug.

Could you give us the planning date when the new plan will be supported?

Regards

**Comment by NingKuang** [comment]
Hi, @yukikashiwabara  could you share more background on your request? What company/project is depending on this, what is your project timeline requirement? Feel free to send the info via email. Thanks.

**Comment by zhongyi-zhang** [comment]
Solved by #204. Planning to release a new tile next week.

**Comment by bruxelles86** [comment]
Hi. I'm a colleague of @yukikashiwabara (we're working for Renault). As you say, it doesn't seem possible at the moment to create/scale to 'general purpose' or 'memory optimized' Postgres databases. This would be a useful feature, as one of our 'basic' service instances in production was overloaded, and a colleague just needed to manually provision a replacement. It would be great to be able to do this with the service broker.
